### DISCUSSION TITLE
how to fix error AttributeError: 'Namespace' object has no attribute 'nowebui'?

### DISCUSSION THREAD

**Discussion by edwardkstam** [TheLastBen/fast-stable-diffusion#180] [Q&A] [answered]
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 159, in <module>
    if cmd_opts.nowebui:
AttributeError: 'Namespace' object has no attribute 'nowebui'

**Comment by TheLastBen** [comment, marked as answer]
check the box "update_repo" in the second cell
